requirements.txt

```diff
@@ -1,3 +1,4 @@
 # Library so -e .
 -e .
+pymesos
```

Comment:
we probably don't want pymesos here, instead in a separate requirements-mesos_plugin.txt:

```
-e .[mesos_executor]
```
Or something. Tbh I'm sorta fine with a separate tox target (see the mesos tox target) that has that dependency there.